src/Controls/src/Core/Layout/Layout.cs

```diff
@@ -137,6 +137,7 @@ public Thickness Padding
 
 		IEnumerator IEnumerable.GetEnumerator() => _children.GetEnumerator();
 
+		[Obsolete("Use Measure without flags argument")]
 		public override SizeRequest Measure(double widthConstraint, double heightConstraint, MeasureFlags flags = MeasureFlags.None)
```

Comment:
Should we be removing the defaults here with this too? I worry that View.Measure will call a different thing than Layout.Measure.

Reply:
Maybe we can just delete this override now that the base does it?